site/src/pages/CreateTemplatePage/BuildLogsDrawer.tsx

```diff
@@ -65,6 +79,28 @@ export const BuildLogsDrawer: FC<BuildLogsDrawerProps> = ({
 					</IconButton>
 				</header>
 
+				{  !logs && !provisionerDaemonsLoading && (
```

Comment:
I think all of this (and probably the query above, too) shoudl be refactored into a new component, similar to how `<MissingVariablesDrawer>` below is.

Reply:
done!